src/Sylius/Bundle/AdminBundle/Resources/config/grids/product.yml

```
@@ -7,7 +7,7 @@ sylius_grid:
                     class: "%sylius.model.product.class%"
                     repository:
                         method: createListQueryBuilder
-                        arguments: ["%locale%", $taxonId]
+                        arguments: ["expr:service('sylius.context.locale.admin_based').getLocaleCode()", $taxonId]
```

What will happen, if we will call:
```suggestion
                        arguments: ["expr:service('sylius.context.locale').getLocaleCode()", $taxonId]
```